zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -97,11 +97,11 @@ service RemoteInterpreterService {
   void open(1: string sessionId, 2: string className) throws (1: InterpreterRPCException ex);
   void close(1: string sessionId, 2: string className) throws (1: InterpreterRPCException ex);
   void reconnect(1: string host, 2: i32 port) throws (1: InterpreterRPCException ex);
-  RemoteInterpreterResult interpret(1: string sessionId, 2: string className, 3: string st, 4: RemoteInterpreterContext interpreterContext) throws (1: InterpreterRPCException ex);
```

Comment:
I think the Thrift files needs to be generated again since new changes to this file